### PR TITLE
spotify support

### PR DESCRIPTION
**Pull request by KiddoKenshin**
Spotify's oembed is undocumented. 
Also, there are some core changes in order for spotify to work.

Please check the inline comment :)

### CODE REVIEW

File: embedo.js [hunk 1]
```diff
@@ -399,7 +406,7 @@
       options = options || {};
       var target = document.head || document.getElementsByTagName('head')[0];
       var script = document.createElement('script');
-      var jsonpCallback = 'embedo_fetch_' + Embedo.utils.uuid();
+      var jsonpCallback = 'embedofetch' + Embedo.utils.uuid();
```
Comment (KiddoKenshin) [inline embedo.js]: Callback request sample: ~~~&callback=embedo_fetch_1nhwnq
However, spotify tends to search for embedofetch1nhwnq().

File: embedo.js [hunk 2]
```diff
@@ -1315,6 +1322,44 @@
...
+    var embed_options = Embedo.utils.merge({
+        url: url
+      },
+      options,
+      Embedo.defaults.RESTRICTED
+    );
+    var embed_uri =
+      Embedo.defaults.SOURCES.spotify.oEmbed + '?' + Embedo.utils.querystring(embed_options);
+
+    Embedo.utils.fetch(embed_uri, function (error, content) {
```
Comment (KiddoKenshin) [inline embedo.js]: I was trying .ajax, however Spotify did not provide proper CORS header upon response and it cause error.

**Comment by shobhitsharma** [comment]
@KiddoKenshin You might not need to do that, because spotify embed URLs are publically allowed to embed. By default embedo's fallback iframe service will embed it. See https://codepen.io/shobhitsharma/pen/prOGzd

**Comment by KiddoKenshin** [comment]
My apologies *blush*
Thank you for pointing it out :)

I close this PR m(_ _)m

**Pull request closed without merging**